examples/programmatic/tensor_basis/options.yaml

```diff
@@ -0,0 +1,56 @@
+device: cuda
+base_precision: 32
+seed: 42
+
+architecture:
+  name: experimental.soap_bpnn
+  model:
+    soap:
+      cutoff: 5.0
+      max_radial: 9
+      max_angular: 5
+      atomic_gaussian_width: 0.3
+      center_atom_weight: 1.0
+      cutoff_function:
+        ShiftedCosine:
+          width: 1.0
+      radial_scaling:
+        Willatt2018:
+          rate: 1.0
+          scale: 2.0
+          exponent: 7.0
+    bpnn:
+      layernorm: true
+      num_hidden_layers: 2
+      num_neurons_per_layer: 64
+    heads: {}
+    zbl: false
+
+  training:
+    learning_rate: 1e-3
+    batch_size: 10
+    distributed: false
+    scheduler_patience: 50
+    scheduler_factor: 0.8
+    log_interval: 1
+    log_mae: true
+    checkpoint_interval: 10
+    num_epochs: 100
```

Comment:
IMO this is too intimidating for a tutorial. I would eliminate all the parameters that you don't change from the defaults (if you take them out, the defaults will be used)